### PR TITLE
I tried again ...

### PR DESCRIPTION
**Pull request by qwertystars**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/qwertystars.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "qwertystars",
+        "email": "srijan.guchhait@gmail.com"
+    },
+    "record": {
+        "CNAME":"qwertystars.000.pe"
```
Comment (21Z) [inline domains/qwertystars.json:7]: Incomplete website I assume?

**Review by 21Z — CHANGES_REQUESTED** [review]

**Review by MaskDuck — CHANGES_REQUESTED** [review]
incompleted website, feel free to return when you have completed

File: domains/qwertystars.json (hunk 1 above)
Comment (qwertystars) [reply domains/qwertystars.json:7]: yes wanting to reserve this domain for temp. will buy new ip address and host of my own in the next few months.

Comment (qwertystars) [reply domains/qwertystars.json:7]: Is it possible to do it or should i try again later?

Comment (21Z) [reply domains/qwertystars.json:7]: I don't think anyone is going to take this domain, You can make another PR again after a few months.

**Comment by qwertystars** [comment]
So, got a webserver running. 
image
it has some directories. 
thinking of making it a mirror site for some open source sites.
will make a blog as well soon.
image
yes it is from bellard.org
ts_ server not running as of now. (trying to figure out why my .bin files not being recognised)
image

**Comment by qwertystars** [comment]
Also do you guys support ns records?
or do i need to do cname?

**Comment by qwertystars** [comment]
the current website is being hosted at qwertystars-windows.serveo.net

**Comment by qwertystars** [comment]
i just read a issue that was closed where u guys discussed about support for NS records. so, u guys don't support it from what i understood.

changing my records to cname.

File: domains/qwertystars.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "qwertystars",
+        "email": "srijan.guchhait@gmail.com"
+    },
+    "record": {
+        "CNAME":"qwertystars-ts-server.serveo.net"
```
Comment (21Z) [inline domains/qwertystars.json:7]: Can't reach website, Error 502, Could you provide a preview of your website?

**Review by 21Z — CHANGES_REQUESTED** [review]

File: domains/qwertystars.json (hunk 2 above)
Comment (qwertystars) [reply domains/qwertystars.json:7]: should be online i again. (unfortunately the current method of making my site available, is well buggy). looking for a more stable way. will request a review later (2-3 months) when site functionality is stable.

**Pull request closed without merging**